Extra args for aws cli

filipef101 opened the pull request:
Accept extra args for aws cli and reduce app icon size to 90

Commit:
Extra args for aws cli

hermanbanken commented on bundle.js:
```diff
@@ -29560,8 +29560,9 @@ run().catch(console.error);
   const bucket = s3name(core.getInput("bucket", { required: true }));
   const destinationPath = core.getInput("destinationPath", { required: false }) || "/";
-  const destinationUrl = core.getInput("destinationUrl", { required: false }) || `https://${bucket}.s3.amazonaws.com${path2.join("/", destinationPath, "/")}`;
+  const destinationUrl = core.getInput("destinationUrl", { required: false }) || `https://${bucket}.storage.googleapis.com${path2.join("/", destinationPath, "/")}`;
```
Somehow this changed in the bundle 😅

filipef101 replied:
Didn't mean to commit that :) forgot to re-run build

hermanbanken reviewed (commented):
Good idea to make it more customizable via extraArgs. Maybe the QR code size should also be customizable... I think the 200 is about right for our _my_ use case, but lets make it configurable then!

filipef101 commented:
> Good idea to make it more customizable via extraArgs. Maybe the QR code size should also be customizable... I think the 200 is about right for our _my_ use case, but lets make it configurable then!

@hermanbanken It was the app icon I reduced, 200 seems quite a lot 😅 , want me to keep as it was and tackle in another PR?

Edit: Changed the size of the wrong icon, updated

Commit:
update bundle.js

Commit:
Fix icon sizes

hermanbanken commented on index.ts:
```diff
@@ -53,7 +54,7 @@ async function run() {
   }
   const remote = `s3://${path.join(bucket, destinationPath)}`;
   console.log(`Syncing ${dest} to ${remote}`);
-  await promisify(exec)(`aws s3 sync ${dest} ${remote} --acl public-read;`);
```
I needed to check how getInput deals with absent values (https://github.com/actions/toolkit/blob/f8a69bc473af4a204d0c03de61d5c9d1300dfb17/packages/core/src/core.ts#L130) luckily it defaults to empty string instead of `undefined` 👍

hermanbanken commented:
Thanks!
